extensions/typescript/package.json

```diff
@@ -106,6 +106,16 @@
           "description": "%typescript.tsserver.log%",
           "scope": "window"
         },
+        "typescript.tsserver.pluginPaths": {
```

Comment:
Add ` "isExecutable": true` here. This prevents this from ever being used as a workspace setting

Reply:
Why do you think it should be a user-only setting?
Specifying pluginPaths on workspace level can be quite useful, especially in conjunction with relative paths. In this case plugins configuration can be committed to a source control and shared between team members.
If this is this because of the security concerns, I don't believe this is a more dangerous option than ability to specify `typescript.tsdk` on a workspace-level.

Reply:
Yes it is for security reasons. Tsdk asks for user confirmation before executing any code in the workspace but that flow is complicated so I don't want to duplicate it here. Instead we should just limit these this to be a user only setting